package.json

```diff
@@ -82,15 +82,15 @@
     "@types/node": "14.14.10",
     "babel-eslint": "10.1.0",
     "babel-loader": "8.2.2",
-    "eslint": "7.25.0",
-    "eslint-config-airbnb": "18.2.0",
+    "eslint": "^7.25.0",
```

Comment:
We prefer to pin specific package versions (npm will annoyingly add these carets automatically oftentimes when running `npm install`)

Reply:
I noticed this too, I'm going to revert these